README.rst

```diff
@@ -29,6 +29,15 @@ You were warned that both stresstests and benchmarks
 put lots of pressure and load on your machine. Be prepared!
 
 
+By default Crate Stresstests and Benchmarks fetch Crate by version, by setting
```

Comment:
By default the stresstests and benchmark use the Crate version specified by the ``crate.version`` system property.
It is also possible to provide a specific Crate version using a URL. The URL is set using the ``crate.url`` system property, e.g.::

    ...